components/frontend-react/src/containers/Dashboard.js

```diff
@@ -58,6 +60,40 @@ const styles = theme => ({
 })
 
 class Dashboard extends React.Component {
+
+  constructor(){
+    super();
+    this.state = {
+      completedList:[],
+      queuedList:[]
+    }
+  }
+
+  componentWillMount() {
+    let token = sessionStorage.getItem('token');
+    axios({
+      method:'get',
+      url: `${process.env.REACT_APP_API_URL}/api/user/downloads/1`,
```

Comment:
Getting complete URL from `process.env`, instead of hard coding endpoint definition here, makes maintain everything under single source easy:) 
LIke `process.env.READCT.. + process.env.API_USER_DOWNLOAD_1`

Reply:
Or maybe, Nirmal. we can add a list of all requests in env file and use them like,
```
REACT_APP_API_GET_DOWNLOADS=http://0.0.0.0:5000/api/user/downloads/1
REACT_APP_API_POST_LOGIN=http://0.0.0.0:5000/api/login
```
I think this one will be better. What are your thoughts?

Reply:
Let's keep these as is, anyway calling API endpoints should go inside Redux Sagas, this is a temporary state we have here.

Reply:
Agreed @agentmilindu